cmd/conformance-tester/pkg/scenarios/aws.go

```diff
@@ -123,6 +123,7 @@ func (s *awsScenario) MachineDeployments(ctx context.Context, num int, secrets t
 			WithAvailabilityZone(*subnet.AvailabilityZone).
 			WithSubnetID(*subnet.SubnetId).
 			WithSpotInstanceMaxPrice("0.5").
+			WithAMI("ami-014dd8ec7f09293e6").
```

Comment:
```suggestion
			WithAMI("ami-0300b7cf57a2ea956").
```

This one gonna work 100%